pkg/apis/v2v/v1alpha1/ovirtresourcemapping_types.go

```diff
@@ -0,0 +1,89 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// OvirtResourceMappingSpec defines the desired state of OvirtResourceMapping
+// +k8s:openapi-gen=true
+type OvirtResourceMappingSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// +optional
+	NetworkMappings *[]ResourceMappingItem `json:"networkMappings,omitempty"`
+
+	// +optional
+	StoragekMappings *[]ResourceMappingItem `json:"storageMappings,omitempty"`
+
+	// +optional
+	AffinityMappings *[]AffinityMappingItem `json:"affinityMappings,omitempty"`
+}
+
+// Source defines how to identify a resource on ovirt, either by ID or by name
+// +k8s:openapi-gen=true
+type Source struct {
+	// +optional
+	Name *string `json:"name,omitempty"`
+
+	// +optional
+	ID *string `json:"id,omitempty"`
+}
+
+// ResourceMappingItem defines the mapping of a single resource from ovirt to kubevirt
+// +k8s:openapi-gen=true
+type ResourceMappingItem struct {
+	Source Source `json:"source"`
+	Target string `json:"target"`
```

Comment:
Would it be always sufficient this to be string? Wouldn't it be better to make it object, so it can be enhanced in future?

Reply:
:+1: Eventually we may need to map more complicated structures.